lib/vector-core/src/event/finalization.rs

```diff
@@ -151,7 +153,7 @@ impl Future for BatchStatusReceiver {
             Poll::Pending => Poll::Pending,
             Poll::Ready(Ok(status)) => Poll::Ready(status),
             Poll::Ready(Err(error)) => {
-                error!(message = "Batch status receiver dropped before sending.", %error);
+                error!(%error, "Batch status receiver dropped before sending.");
```

Comment:
Is there a reason for this change?

Reply:
Just shorter.  If you put all of the structured fields first, you can put the string literal message as the last argument to the `tracing` macros without having to do `message = ".."`, etc.